content/index.md

```diff
@@ -41,19 +41,25 @@ This resource provides a variety of modules to help you create courses on digita
 
 ## Overview
 
+This curricula resource provides material for teaching accessibility, and for reviewing accessibility courses. You can use it to develop robust digital accessibility courses. You can use it to include shorter accessibility topics in other courses, such as programming or graphics design.
+
+You can also use this resource to evaluate existing or proposed courses.
+
+This resource includes modules that cover accessibility foundations that apply broadly, and modules specific for developers, designers, and content authors. The Foundation and Developing modules are available now. The Designer and Author modules will be available in 2021.
```

Comment:
```suggestion
This resource includes modules that cover accessibility foundations that apply broadly, and modules specific for development, designning, and content authoring. The Foundation and Developer modules are available now. The Designer and Author modules will be available in 2021.
```

Reply:
+1 to type of tasks and activities, rather than roles

How about:
> ...modules specific to developing, designing, and content authoring.

(Instead of what you have: ... modules specific for development, designning, and content authoring.)

(here and in summary previous -- sorry I read too fast and missed it there.)

Reply:
I updated this in the pull request, so we can move forward if you agree.